app-emulation/atari800_libretro/atari800_libretro-3.1.0.20171023git.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="An Atari 8-bit computer and 5200 console emulator core for libretro"
+DESCRIPTION="This is a free and portable Atari 8-bit computer and 5200 console \
+emulator, originally written by David Firth and now developed by the Atari800 \
+Development Team."
+HOMEPAGE="https://atari800.github.io/"
+COPYRIGHT="2001-2018, the Atari800 Dev team, the libretro team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="f892b4437d52a3e1c4b42b18f3edc564a1d1045b"
+SOURCE_URI="https://github.com/libretro/libretro-atari800/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="0c412610237e10b518373b5fd0f9bfe743ae03b288122f899887ea745905949a"
+SOURCE_FILENAME="libretro-atari800-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="libretro-atari800-$srcGitRev"
+ADDITIONAL_FILES="atari800_libretro.info.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	atari800_libretro$secondaryArchSuffix = $portVersion
+	addon:atari800_libretro$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	retroarch$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	zlib${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	sed -e "s/@DISPLAY_VERSION@/v${portVersion%git}/" \
```

Comment:
```diff
-	sed -e "s/@DISPLAY_VERSION@/v${portVersion%git}/" \
+	sed -e "s/@DISPLAY_VERSION@/v${portVersion/_/-}-${srcGitRev:0:7}/" \
```
(Do we insert a `-${srcGitRev:0:7}` here? Or is it not necessary?)

Reply:
Not necessary, the one inserted after make is a simple fix to help avoiding a build error without having to use PATCH()